doc/specs/stdlib_strings.md

```diff
@@ -199,25 +199,21 @@ end program demo
 
 #### Description
 
-Extracts the characters from the defined region of the input string by taking strides.
+Extracts the characters from the defined region of the input string by taking strides.  
+Argument `first` and `last` defines this region for extraction by function `slice`.  
+Argument `stride` defines the magnitude and direction (+/-) of stride to be taken while extraction. 
+`stride` when given invalid value 0, is converted to +1.
 
-Deduction Process:
+Deduction Process:  
```

Comment:
```suggestion
Deduction Process:
```

Reply:
When viewed on GitHub the modified `stdlib_strings.md` file looks like [this](https://github.com/Aman-Godara/stdlib/blob/replace_all/doc/specs/stdlib_strings.md#slice).

Reply:
Should I consider this as resolved or needing changes? @jvdp1

Reply:
It was to remove trailing spaces. If there are needed, consider it as resolved. Otherwise, remove them, and it will be resolved too ;)